NorthstarDLL/mods/modmanager.cpp

```diff
@@ -298,11 +298,18 @@ template <ScriptContext context> auto ModConCommandCallback_Internal(std::string
 {
 	if (g_pSquirrel<context>->m_pSQVM && g_pSquirrel<context>->m_pSQVM)
 	{
-		std::vector<std::string> args;
-		args.reserve(command.ArgC());
-		for (int i = 1; i < command.ArgC(); i++)
-			args.push_back(command.Arg(i));
-		g_pSquirrel<context>->AsyncCall(name, args);
+		if (command.ArgC() == 1)
+		{
+			g_pSquirrel<context>->AsyncCall(name);
+		}
+		else
+		{
+			std::vector<std::string> args;
+			args.reserve(command.ArgC());
+			for (int i = 1; i < command.ArgC(); i++)
```

Comment:
> @Glacir: BTW: args passed to squirrel is always a array of string so command.ArgC() is always 2 if more than 0 arg is inputted in console. Not the point of this pr but should we change it?

If it means that this doesn't actually work, then yes.

Reply:
It works but not intuitive. For modders who register concommand, what they expect is register a concommand like
```
command [string a] [string b]
```
and link it to a squirrel function

```
void function foo(string a, string b)
```

But the current logic in ns launcher pass the args as an `array<string>` so modders need to write their squirrel function like this:

```
void function foo(array<string> args)
```

Reply:
Ah, I suppose I misunderstood what you meant. That makes sense.